test/clients/firewall_auto_test.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require_relative "../../src/clients/firewall_auto.rb"
```

Comment:
then I really suggest to not use require_relative and instead use WFM.CallFunction https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/wfm.rb#L164 with given parameters as it is much closer to expected usage. So instead of stubbing args you call `expect(Yast::WFM.CallFunction("firewall_auto", ["Summary"])`